src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/clients/ExternalAPICall.kt

```diff
@@ -0,0 +1,29 @@
+package com.gamingbacklog.api.gamingbacklogapi.clients
+
+import org.springframework.context.annotation.Bean
+import org.springframework.stereotype.Component
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+@Component
+class ExternalAPICall {
+  fun postExternalCall(
```

Comment:
If this is meant to just be a generic client (that is used by IGDBClient as below and might be used for other third party calls in the future), maybe adding "Util" or something like that to the class name/package could make that more clear. 

We might also want to think about being more consistent with our naming i.e. if we have IGDBClient, do we want this to be `ExternalClient`?